tests/e2e/Services/Functions/FunctionsCustomServerTest.php

```diff
@@ -191,8 +191,8 @@ public function testList(array $data): array
             'runtime' => 'php-8.0',
             'entrypoint' => 'index.php',
             'events' => [
-                'users.*.create',
-                'users.*.delete',
+                'buckets.*.create',
+                'buckets.*.delete',
```

Comment:
why did we change the event type here?

Reply:
Below I added actual event-triggered test. Those are just sample values for array attribute.

my test below was failing because of those having user creation listener.

Reply:
I would need to rename those, or increase timeout to wait for more execution to finish

Reply:
Alright 👍